shadows/framework/src/main/java/org/robolectric/shadows/ShadowPlayerBase.java

```diff
@@ -1,12 +1,13 @@
 package org.robolectric.shadows;
 
 import android.media.IAudioService;
-import android.media.PlayerBase;
+import android.os.Build;
 import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
 import org.robolectric.util.ReflectionHelpers;
 
-@Implements(value = PlayerBase.class, isInAndroidSdk = false)
+@Implements(className = "android.media.PlayerBase", isInAndroidSdk = false,
```

Comment:
Remind me why these had to be specified as Strings? Loaded early outside of the sandbox?

Reply:
`PlayerBase` doesn't exist before N I think, so it's a missing class in sandboxes pre N. Projects that only use the android-all dependency would have trouble running tests on older SDKs. I think.